src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -298,6 +298,8 @@ public object Initialize(JToken arg)
             this.LogToWindow($"workspace folder: {this.workspaceFolder ?? "(Null)"}", MessageType.Info);
             this.fileWatcher.ListenAsync(this.workspaceFolder, true, dict => this.InitializeWorkspaceAsync(dict), "*.csproj", "*.dll", "*.qs").Wait(); // not strictly necessary to wait but less confusing
 
+            this.editorState.CreateNotebookProjectAsync().Wait();
```

Comment:
Do we want to always create a notebook project or wait until the first notebook cell?